src/Group.php

```diff
@@ -119,48 +119,71 @@ public function cleanDBonPurge()
         Rule::cleanForItemCriteria($this, '_groups_id%');
     }
 
-
     public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     {
+        /**  @var \DBmysql $DB
+         * @var array $CFG_GLPI
+         */
+        global $DB, $CFG_GLPI;
 
-        if (!$withtemplate && self::canView()) {
-            $nb = 0;
-            switch ($item->getType()) {
-                case 'Group':
-                    /** @var Group $item */
-                    $ong = [];
-                    if ($_SESSION['glpishow_count_on_tabs']) {
-                        $nb = countElementsInTable(
-                            $this->getTable(),
-                            ['groups_id' => $item->getID()]
-                        );
-                    }
-                    $ong[4] = self::createTabEntry(__('Child groups'), $nb);
+        if ($withtemplate || !self::canView()) {
+            return '';
+        }
+
+        $ong = [];
+        switch (get_class($item)) {
+            case Group::class:
+                $nb = 0;
+                if ($_SESSION['glpishow_count_on_tabs']) {
+                    $nb = countElementsInTable(
+                        $this->getTable(),
+                        ['groups_id' => $item->getID()]
+                    );
+                }
+                $ong[4] = self::createTabEntry(__('Child groups'), $nb);
 
+                if ($_SESSION['glpishow_count_on_tabs']) {
+                    if ($item->getField('is_itemgroup')) {
```

Comment:
`CommonDBTM::getField()`  must be used only to display values.

```suggestion
                    if ($item->fields['is_itemgroup']) {
```